client/src/js/components/dateEditor.js

```diff
@@ -0,0 +1,46 @@
+/**
+* The date editor component
+*
+* @module components/dateEditor
+*
+* @description a component to deal with date, it let you choose a date
+* This component can do his own validation based on parameters values provided*
+* 
+**/ 
+
+/**
+* A date editor component controller
```

Comment:
Instead of documenting the injects here, it would be more useful to show an example usage.